tests/Unit/Framework/ConfigTest.php

```diff
@@ -10,38 +10,25 @@
 
 final class ConfigTest extends TestCase
 {
-    private Config $config;
-
-    public function setUp(): void
-    {
-        Config::resetInstance();
-        $this->config = Config::getInstance();
-    }
-
-    public function tearDown(): void
-    {
-        Config::resetInstance();
-    }
-
     public function test_get_undefined_key(): void
     {
-        $this->expectExceptionMessageMatches('/Could not find config key "key"/');
-        $this->config->get('key');
+        $this->expectExceptionMessageMatches('/Could not find config key "undefined-key"/');
+        (Config::getInstance())->get('undefined-key');
```

Comment:
There is no need for the parenthesis here, you can do (and I would prefer this too):
```php
Config::getInstance()->get('undefined-key');
```